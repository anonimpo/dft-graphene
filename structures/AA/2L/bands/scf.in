&CONTROL
calculation   = 'scf'
pseudo_dir    = '/home/rfa/Skripsi/DFT/dft-graphene/pseudo'
outdir        = '/home/rfa/Skripsi/DFT/dft-graphene/tmp'
prefix        = 'gr-aa-2L'
/
&SYSTEM
ibrav         = 4
a             = 2.4639055825
c             = 15.0
nat           = 4
ntyp          = 1
occupations   = 'smearing'
smearing      = 'mv'
degauss       = 0.020
ecutwfc       = 60  ! Use converged value from tests
/
&ELECTRONS
mixing_beta   = 0.7
conv_thr      = 1.0D-8
/
ATOMIC_SPECIES
C 12.0107 C.pbe-n-rrkjus_psl.0.1.UPF
ATOMIC_POSITIONS (crystal)
C  0.333333333  0.666666666  0.4823607596  ! First layer modify after relaxation in z axis
C  0.666666666  0.333333333  0.4823607596  ! First layer 
C  0.333333333  0.666666666  0.5509725734  ! Second layer (AA stacking)
C  0.666666666  0.333333333  0.5509725734  ! Second layer (AA stacking)
K_POINTS (automatic)
12 12 1 0 0 0
