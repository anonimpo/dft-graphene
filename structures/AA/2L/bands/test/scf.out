
     Program PWSCF v.7.4.1 starts on 19Mar2025 at  4:26:16 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
         "P. Giannozzi et al., J. Chem. Phys. 152 154105 (2020);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on     7 processors

     MPI processes distributed on     1 nodes
     3544 MiB available memory on the printing compute node when the environment starts

     Waiting for input...
     Reading input from standard input

     Current dimensions of program PWSCF are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  4

     R & G space division:  proc/nbgrp/npool/nimage =       7
     Subspace diagonalization in iterative solution of the eigenvalue problem:
     a serial algorithm will be used


     Parallelization info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Min          51      51     15                 4783     4783     815
     Max          52      52     16                 4800     4800     838
     Sum         361     361    109                33523    33523    5791

     Using Slab Decomposition



     bravais-lattice index     =            4
     lattice parameter (alat)  =       4.6561  a.u.
     unit-cell volume          =     532.1899 (a.u.)^3
     number of atoms/cell      =            4
     number of atomic types    =            1
     number of electrons       =        16.00
     number of Kohn-Sham states=           12
     kinetic-energy cutoff     =      60.0000  Ry
     charge density cutoff     =     240.0000  Ry
     scf convergence threshold =      1.0E-06
     mixing beta               =       0.7000
     number of iterations used =            8  plain     mixing
     Exchange-correlation= PBE
                           (   1   4   3   4   0   0   0)

     celldm(1)=   4.656107  celldm(2)=   0.000000  celldm(3)=   6.087896
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (   1.000000   0.000000   0.000000 )  
               a(2) = (  -0.500000   0.866025   0.000000 )  
               a(3) = (   0.000000   0.000000   6.087896 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = (  1.000000  0.577350 -0.000000 )  
               b(2) = (  0.000000  1.154701  0.000000 )  
               b(3) = (  0.000000 -0.000000  0.164260 )  


     PseudoPot. # 1 for C  read from file:
     ../../../../pseudo/C.pbe-n-rrkjus_psl.0.1.UPF
     MD5 check sum: 96a6a315177d77e6c95bf6fd3870f9e3
     Pseudo is Ultrasoft + core correction, Zval =  4.0
     Generated using &quot;atomic&quot; code by A. Dal Corso  v.6.3MaX
     Using radial grid of 1073 points,  4 beta functions with: 
                l(1) =   0
                l(2) =   0
                l(3) =   1
                l(4) =   1
     Q(r) pseudized with 0 coefficients 


     atomic species   valence    mass     pseudopotential
     C                 4.00    12.01070     C ( 1.00)

     12 Sym. Ops. (no inversion) found



   Cartesian axes

     site n.     atom                  positions (alat units)
         1        C      tau(   1) = (   0.0000000   0.5773503   3.0439478  )
         2        C      tau(   2) = (   0.5000000   0.2886751   3.0439478  )
         3        C      tau(   3) = (   0.0000000   0.5773503   3.2468777  )
         4        C      tau(   4) = (   0.5000000   0.2886751   3.2468777  )

     number of k points=    19  Marzari-Vanderbilt smearing, width (Ry)=  0.0200
                       cart. coord. in units 2pi/alat
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   0.0138889
        k(    2) = (   0.0000000   0.0962250   0.0000000), wk =   0.0833333
        k(    3) = (   0.0000000   0.1924501   0.0000000), wk =   0.0833333
        k(    4) = (   0.0000000   0.2886751   0.0000000), wk =   0.0833333
        k(    5) = (   0.0000000   0.3849002   0.0000000), wk =   0.0833333
        k(    6) = (   0.0000000   0.4811252   0.0000000), wk =   0.0833333
        k(    7) = (   0.0000000  -0.5773503   0.0000000), wk =   0.0416667
        k(    8) = (   0.0833333   0.1443376   0.0000000), wk =   0.0833333
        k(    9) = (   0.0833333   0.2405626   0.0000000), wk =   0.1666667
        k(   10) = (   0.0833333   0.3367877   0.0000000), wk =   0.1666667
        k(   11) = (   0.0833333   0.4330127   0.0000000), wk =   0.1666667
        k(   12) = (   0.0833333   0.5292377   0.0000000), wk =   0.1666667
        k(   13) = (   0.1666667   0.2886751   0.0000000), wk =   0.0833333
        k(   14) = (   0.1666667   0.3849002   0.0000000), wk =   0.1666667
        k(   15) = (   0.1666667   0.4811252   0.0000000), wk =   0.1666667
        k(   16) = (   0.1666667   0.5773503   0.0000000), wk =   0.0833333
        k(   17) = (   0.2500000   0.4330127   0.0000000), wk =   0.0833333
        k(   18) = (   0.2500000   0.5292377   0.0000000), wk =   0.1666667
        k(   19) = (   0.3333333   0.5773503   0.0000000), wk =   0.0277778

     Dense  grid:    33523 G-vectors     FFT dimensions: (  24,  24, 144)

     Estimated max dynamical RAM per process >       9.08 MB

     Estimated total dynamical RAM >      63.55 MB

     Check: negative core charge=   -0.000004

     Initial potential from superposition of free atoms

     starting charge      15.9998, renormalised to      16.0000

     negative rho (up, down):  1.235E-04 0.000E+00
     Starting wfcs are   16 randomized atomic wfcs

     total cpu time spent up to now is        0.8 secs

     per-process dynamical memory:    12.1 Mb

     Self-consistent Calculation

     iteration #  1     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations =  8.4

     negative rho (up, down):  1.233E-03 0.000E+00

     total cpu time spent up to now is        1.6 secs

     total energy              =     -22.63914063 Ry
     estimated scf accuracy    <       9.84287862 Ry

     iteration #  2     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations =  9.6

     total cpu time spent up to now is        2.4 secs

     total energy              =     -23.53856566 Ry
     estimated scf accuracy    <       7.05421785 Ry

     iteration #  3     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations =  5.4

     total cpu time spent up to now is        2.8 secs

     total energy              =     -26.13326232 Ry
     estimated scf accuracy    <       0.42906157 Ry

     iteration #  4     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  2.68E-03,  avg # of iterations =  4.1

     total cpu time spent up to now is        3.2 secs

     total energy              =     -26.11482188 Ry
     estimated scf accuracy    <       0.20028819 Ry

     iteration #  5     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.25E-03,  avg # of iterations =  5.3

     negative rho (up, down):  2.698E-08 0.000E+00

     total cpu time spent up to now is        3.6 secs

     total energy              =     -26.10425839 Ry
     estimated scf accuracy    <       0.06005878 Ry

     iteration #  6     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  3.75E-04,  avg # of iterations =  8.9

     total cpu time spent up to now is        4.2 secs

     total energy              =     -26.12937824 Ry
     estimated scf accuracy    <       0.00628909 Ry

     iteration #  7     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     c_bands:  1 eigenvalues not converged
     c_bands:  1 eigenvalues not converged
     ethr =  3.93E-05,  avg # of iterations = 10.4

     negative rho (up, down):  4.325E-06 0.000E+00

     total cpu time spent up to now is        4.9 secs

     total energy              =     -26.13880132 Ry
     estimated scf accuracy    <       0.01102132 Ry

     iteration #  8     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     c_bands:  1 eigenvalues not converged
     ethr =  3.93E-05,  avg # of iterations =  7.3

     negative rho (up, down):  1.762E-03 0.000E+00

     total cpu time spent up to now is        5.4 secs

     total energy              =     -26.14109842 Ry
     estimated scf accuracy    <       0.02097421 Ry

     iteration #  9     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  3.93E-05,  avg # of iterations =  5.1

     negative rho (up, down):  4.267E-04 0.000E+00

     total cpu time spent up to now is        5.8 secs

     total energy              =     -26.13715406 Ry
     estimated scf accuracy    <       0.02428610 Ry

     iteration # 10     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  3.93E-05,  avg # of iterations =  1.1

     negative rho (up, down):  5.302E-04 0.000E+00

     total cpu time spent up to now is        6.1 secs

     total energy              =     -26.13868326 Ry
     estimated scf accuracy    <       0.01512740 Ry

     iteration # 11     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  3.93E-05,  avg # of iterations =  1.0

     negative rho (up, down):  9.523E-04 0.000E+00

     total cpu time spent up to now is        6.3 secs

     total energy              =     -26.13662000 Ry
     estimated scf accuracy    <       0.01830140 Ry

     iteration # 12     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  3.93E-05,  avg # of iterations =  1.1

     negative rho (up, down):  1.044E-03 0.000E+00

     total cpu time spent up to now is        6.6 secs

     total energy              =     -26.13285862 Ry
     estimated scf accuracy    <       0.01294065 Ry

     iteration # 13     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  3.93E-05,  avg # of iterations =  1.6

     negative rho (up, down):  1.064E-03 0.000E+00

     total cpu time spent up to now is        6.9 secs

     total energy              =     -26.13286511 Ry
     estimated scf accuracy    <       0.00408260 Ry

     iteration # 14     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  2.55E-05,  avg # of iterations =  2.5

     negative rho (up, down):  1.067E-03 0.000E+00

     total cpu time spent up to now is        7.2 secs

     total energy              =     -26.13306626 Ry
     estimated scf accuracy    <       0.00249957 Ry

     iteration # 15     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.56E-05,  avg # of iterations =  2.4

     negative rho (up, down):  1.164E-03 0.000E+00

     total cpu time spent up to now is        7.5 secs

     total energy              =     -26.13195276 Ry
     estimated scf accuracy    <       0.00190963 Ry

     iteration # 16     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     c_bands:  1 eigenvalues not converged
     ethr =  1.19E-05,  avg # of iterations =  7.8

     negative rho (up, down):  1.167E-03 0.000E+00

     total cpu time spent up to now is        8.1 secs

     total energy              =     -26.13345954 Ry
     estimated scf accuracy    <       0.00076130 Ry

     iteration # 17     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  4.76E-06,  avg # of iterations =  5.0

     negative rho (up, down):  1.126E-03 0.000E+00

     total cpu time spent up to now is        8.5 secs

     total energy              =     -26.13357615 Ry
     estimated scf accuracy    <       0.00109379 Ry

     iteration # 18     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  4.76E-06,  avg # of iterations =  1.3

     negative rho (up, down):  1.088E-03 0.000E+00

     total cpu time spent up to now is        8.8 secs

     total energy              =     -26.13336055 Ry
     estimated scf accuracy    <       0.00117368 Ry

     iteration # 19     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  4.76E-06,  avg # of iterations =  1.0

     negative rho (up, down):  1.095E-03 0.000E+00

     total cpu time spent up to now is        9.0 secs

     total energy              =     -26.13326040 Ry
     estimated scf accuracy    <       0.00065023 Ry

     iteration # 20     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  4.06E-06,  avg # of iterations =  1.1

     negative rho (up, down):  1.109E-03 0.000E+00

     total cpu time spent up to now is        9.3 secs

     total energy              =     -26.13327714 Ry
     estimated scf accuracy    <       0.00035407 Ry

     iteration # 21     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  2.21E-06,  avg # of iterations =  1.1

     negative rho (up, down):  1.112E-03 0.000E+00

     total cpu time spent up to now is        9.6 secs

     total energy              =     -26.13335004 Ry
     estimated scf accuracy    <       0.00034800 Ry

     iteration # 22     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  2.17E-06,  avg # of iterations =  3.9

     negative rho (up, down):  1.055E-03 0.000E+00

     total cpu time spent up to now is       10.0 secs

     total energy              =     -26.13328630 Ry
     estimated scf accuracy    <       0.00063967 Ry

     iteration # 23     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  2.17E-06,  avg # of iterations =  1.6

     negative rho (up, down):  1.030E-03 0.000E+00

     total cpu time spent up to now is       10.2 secs

     total energy              =     -26.13313506 Ry
     estimated scf accuracy    <       0.00034966 Ry

     iteration # 24     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  2.17E-06,  avg # of iterations =  1.8

     negative rho (up, down):  1.054E-03 0.000E+00

     total cpu time spent up to now is       10.5 secs

     total energy              =     -26.13317575 Ry
     estimated scf accuracy    <       0.00002426 Ry

     iteration # 25     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.52E-07,  avg # of iterations =  5.4

     negative rho (up, down):  1.050E-03 0.000E+00

     total cpu time spent up to now is       11.0 secs

     total energy              =     -26.13319337 Ry
     estimated scf accuracy    <       0.00000506 Ry

     iteration # 26     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  3.16E-08,  avg # of iterations =  4.9

     negative rho (up, down):  1.056E-03 0.000E+00

     total cpu time spent up to now is       11.5 secs

     End of self-consistent calculation

          k = 0.0000 0.0000 0.0000 (  4149 PWs)   bands (ev):

   -24.5073 -23.6714 -23.6714 -16.2461 -10.9787 -10.9787  -5.1590   2.7527
     3.3727   3.4673   4.7341   5.6149

          k = 0.0000 0.0962 0.0000 (  4170 PWs)   bands (ev):

   -24.6211 -23.8720 -23.7629 -16.0686 -10.5558 -10.4039  -4.9995   1.9597
     3.5953   3.6934   4.9541   5.6064

          k = 0.0000 0.1925 0.0000 (  4194 PWs)   bands (ev):

   -25.1265 -24.3731 -23.7327 -15.5401  -9.4205  -8.8492  -4.5258   0.0827
     4.2606   4.3706   5.1445   5.6129

          k = 0.0000 0.2887 0.0000 (  4186 PWs)   bands (ev):

   -25.8604 -24.9718 -23.5001 -14.6781  -7.8825  -6.6514  -3.7451  -2.2075
     4.3741   5.3575   5.4991   6.7036

          k = 0.0000 0.3849 0.0000 (  4180 PWs)   bands (ev):

   -26.5232 -25.4987 -23.2171 -13.5287  -6.3117  -4.5251  -4.1925  -2.5990
     3.3263   6.8393   7.0615   8.1889

          k = 0.0000 0.4811 0.0000 (  4172 PWs)   bands (ev):

   -26.9648 -25.8492 -22.9946 -12.2553  -6.5546  -5.1124  -2.3137  -0.5888
     2.1774   5.4419   8.1219   8.3716

          k = 0.0000-0.5774 0.0000 (  4184 PWs)   bands (ev):

   -27.1187 -25.9718 -22.9110 -11.5150  -7.5556  -4.6576  -1.5997   1.3344
     1.9654   2.9212   7.9452   8.8292

          k = 0.0833 0.1443 0.0000 (  4169 PWs)   bands (ev):

   -24.9966 -24.0201 -23.9247 -15.7147  -9.7553  -9.3640  -4.6830   0.6608
     4.0377   4.1452   5.3003   5.3932

          k = 0.0833 0.2406 0.0000 (  4180 PWs)   bands (ev):

   -25.7187 -24.4793 -23.7606 -15.0178  -8.3764  -7.5446  -4.0578  -1.3296
     4.6931   4.9068   5.0479   6.2671

          k = 0.0833 0.3368 0.0000 (  4175 PWs)   bands (ev):

   -26.4533 -25.0008 -23.4765 -14.0067  -6.7774  -5.2528  -3.5464  -3.1179
     3.7978   6.0844   6.3931   7.5562

          k = 0.0833 0.4330 0.0000 (  4187 PWs)   bands (ev):

   -27.0147 -25.4188 -23.2084 -12.7627  -5.6909  -5.3460  -3.0026  -1.6479
     2.6836   6.2602   6.8161   8.2658

          k = 0.0833 0.5292 0.0000 (  4195 PWs)   bands (ev):

   -27.3141 -25.6466 -23.0469 -11.6003  -7.3329  -4.4817  -1.6887   0.7479
     1.7056   3.9371   5.8926   8.8979

          k = 0.1667 0.2887 0.0000 (  4170 PWs)   bands (ev):

   -26.4865 -24.5600 -23.7692 -14.1677  -6.8466  -5.7292  -3.2834  -3.1298
     3.9767   5.7353   6.1675   6.9709

          k = 0.1667 0.3849 0.0000 (  4172 PWs)   bands (ev):

   -27.1438 -24.8124 -23.5923 -13.0344  -5.2519  -5.0631  -3.6509  -2.1465
     3.0240   4.9941   7.6864   7.8476

          k = 0.1667 0.4811 0.0000 (  4170 PWs)   bands (ev):

   -27.5730 -25.0398 -23.4052 -11.7497  -6.8780  -4.0340  -1.9181  -0.3004
     1.9500   4.1321   5.1324   8.2240

          k = 0.1667 0.5774 0.0000 (  4178 PWs)   bands (ev):

   -27.7219 -25.1257 -23.3301 -10.9608  -7.8539  -3.5710  -1.1581   1.2472
     1.8172   2.9054   3.8332   7.8903

          k = 0.2500 0.4330 0.0000 (  4156 PWs)   bands (ev):

   -27.6727 -24.6509 -23.6997 -11.8194  -6.6415  -3.7122  -2.1202  -0.7287
     2.0905   3.4662   5.6985   7.8254

          k = 0.2500 0.5292 0.0000 (  4161 PWs)   bands (ev):

   -27.9542 -24.5873 -23.7328 -10.5671  -8.0756  -2.7222  -0.8828   1.0785
     1.1649   2.6525   3.5707   6.8924

          k = 0.3333 0.5774 0.0000 (  4152 PWs)   bands (ev):

   -28.0871 -24.1382 -24.1382  -9.3120  -9.3120  -2.0945   0.1577   0.1577
     2.1268   2.3301   2.3301   6.2163

     the Fermi energy is     1.3376 ev

!    total energy              =     -26.13319659 Ry
     estimated scf accuracy    <       0.00000065 Ry
     smearing contrib. (-TS)   =       0.00008796 Ry
     internal energy E=F+TS    =     -26.13328454 Ry

     The total energy is F=E-TS. E is the sum of the following terms:
     one-electron contribution =    -631.36115841 Ry
     hartree contribution      =     320.08390314 Ry
     xc contribution           =     -21.23348833 Ry
     ewald contribution        =     306.37745906 Ry

     convergence has been achieved in  26 iterations

     Writing all to output data dir ../../../../tmp/gr-aa-2L.save/ :
     XML data file, charge density, pseudopotentials, collected wavefunctions

     init_run     :      0.57s CPU      0.78s WALL (       1 calls)
     electrons    :     18.38s CPU     10.68s WALL (       1 calls)

     Called by init_run:
     wfcinit      :      0.23s CPU      0.13s WALL (       1 calls)
     potinit      :      0.00s CPU      0.01s WALL (       1 calls)
     hinit0       :      0.31s CPU      0.62s WALL (       1 calls)

     Called by electrons:
     c_bands      :     15.44s CPU      8.86s WALL (      26 calls)
     sum_band     :      2.44s CPU      1.41s WALL (      26 calls)
     v_of_rho     :      0.18s CPU      0.17s WALL (      27 calls)
     newd         :      0.16s CPU      0.15s WALL (      27 calls)
     mix_rho      :      0.14s CPU      0.08s WALL (      26 calls)

     Called by c_bands:
     init_us_2    :      0.40s CPU      0.21s WALL (    1007 calls)
     cegterg      :     14.85s CPU      8.54s WALL (     494 calls)

     Called by *egterg:
     cdiaghg      :      1.03s CPU      0.58s WALL (    2566 calls)
     h_psi        :     11.61s CPU      6.69s WALL (    2585 calls)
     s_psi        :      0.29s CPU      0.17s WALL (    2585 calls)
     g_psi        :      0.10s CPU      0.06s WALL (    2072 calls)

     Called by h_psi:
     h_psi:calbec :      1.06s CPU      0.61s WALL (    2585 calls)
     vloc_psi     :      9.94s CPU      5.71s WALL (    2585 calls)
     add_vuspsi   :      0.46s CPU      0.27s WALL (    2585 calls)

     General routines
     calbec       :      1.22s CPU      0.71s WALL (    3079 calls)
     fft          :      0.35s CPU      0.66s WALL (     297 calls)
     ffts         :      0.01s CPU      0.01s WALL (      26 calls)
     fftw         :      9.88s CPU      5.71s WALL (   40936 calls)

     Parallel routines

     PWSCF        :     19.01s CPU     11.55s WALL


   This run was terminated on:   4:26:28  19Mar2025            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
